torchio/transforms/preprocessing/spatial/crop_or_pad.py

```diff
@@ -167,10 +167,19 @@ def _compute_mask_center_crop_or_pad(self, sample: dict):
                 f' not found in sample keys: {tuple(sample.keys())}'
                 f'. Using the center of the volume for cropping/padding instead'
             )
-            warnings.warn(message)
+            warnings.warn(message=message)
```

Comment:
Why this replacement?

Reply:
No particular reason, I just tend to use kwargs instead of args. Sorry about that